src/main/java/org/opentripplanner/gtfs/mapping/FareAttributeMapper.java

```diff
@@ -23,17 +26,24 @@ FareAttribute map(org.onebusaway.gtfs.model.FareAttribute orginal) {
   }
 
   private FareAttribute doMap(org.onebusaway.gtfs.model.FareAttribute rhs) {
-    FareAttribute lhs = new FareAttribute(mapAgencyAndId(rhs.getId()));
-
-    lhs.setPrice(rhs.getPrice());
-    lhs.setCurrencyType(rhs.getCurrencyType());
-    lhs.setPaymentMethod(rhs.getPaymentMethod());
-    lhs.setTransfers(rhs.getTransfers());
-    lhs.setTransferDuration(rhs.getTransferDuration());
-    lhs.setYouthPrice(rhs.getYouthPrice());
-    lhs.setSeniorPrice(rhs.getSeniorPrice());
-    lhs.setJourneyDuration(rhs.getJourneyDuration());
-
-    return lhs;
+    FareAttributeBuilder builder = FareAttribute
+      .of(mapAgencyAndId(rhs.getId()))
+      .setPrice(rhs.getPrice())
+      .setCurrencyType(rhs.getCurrencyType())
+      .setPaymentMethod(rhs.getPaymentMethod())
+      .setYouthPrice(rhs.getYouthPrice())
+      .setSeniorPrice(rhs.getSeniorPrice());
+
+    if (rhs.getTransfers() != MISSING_VALUE) {
```

Comment:
Use the `rhs.isTransfersSet()` not the constant `MISSING_VALUE `. This apply the checks below as well.

Reply:
Done